### DISCUSSION TITLE
51159 wifi module? (as opposed to 50350)

### DISCUSSION THREAD

**Discussion by Jieper001** [plmilord/Hass.io-custom-component-spaclient#23] [General]
Hi,

I've set up a Home Assistant instance just for me to be able to control my spa and later do some integrations to try to have better energy management.

My spa at current does not have the Balboa wifi module.  I am too much of a noob to go the DIY route ([RS485 to wifi](https://community.home-assistant.io/t/balboa-hot-tub-spa-automation-and-power-savings/353032)) to replace the "out of the box" wifi module, so most likely will fork out the +- 300EUR for this bwa wifi module.

My question is this: I see the there is also a 51159 wifi module (as opposed to the 50350) one.  Any pointers if this one works too?  If I order a wifi module here in Europe, it is unsure which one I'll get, hence my question.

Thank you soooo much!

**Comment by plmilord** [comment]
hmmm...don't know anything about 51159. After some googling it seems to be the same module!?

You should be looking for a Balboa BP Wifi Module 50350 in the latest version 50350-06 backwards compatible with BP control systems.

For sure, if your spa is equipped with a Balboa systems, the 50350 module (with this Home Assistant custom, component ;-) ) will change the way you control your spa!

**Reply by Jieper001** [reply]
Thanks!

**Comment by Jieper001** [comment]
Quick update: ordered BWA module that was online advertised as 51159.  Installed and working without any problem - thank you so much for the great work.

**Reply by plmilord** [reply]
Good news! If you have any questions, ideas, feel free to share!
